[2025-02-28T05:20:06.006+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T05:15:00+00:00 [queued]>
[2025-02-28T05:20:06.018+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T05:15:00+00:00 [queued]>
[2025-02-28T05:20:06.019+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-28T05:20:06.040+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-02-28 05:15:00+00:00
[2025-02-28T05:20:06.048+0000] {standard_task_runner.py:60} INFO - Started process 2427 to run task
[2025-02-28T05:20:06.054+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-02-28T05:15:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpm5befmb2']
[2025-02-28T05:20:06.060+0000] {standard_task_runner.py:88} INFO - Job 54: Subtask load_to_postgres_file
[2025-02-28T05:20:06.132+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T05:15:00+00:00 [running]> on host 335ace54b661
[2025-02-28T05:20:06.232+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T05:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T05:15:00+00:00'
[2025-02-28T05:20:06.235+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T05:20:06.237+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-02-28T05:20:06.252+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T05:20:07.573+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/12-20-03.csv' từ MinIO
[2025-02-28T05:20:07.574+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-02-28T05:20:07.575+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-02-28T05:20:07.653+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T05:20:07.687+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250228T051500, start_date=20250228T052006, end_date=20250228T052007
[2025-02-28T05:20:07.716+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-28T05:20:07.738+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
